test/lib/tasks/permissions_promoter_test.rb

```diff
@@ -25,8 +25,8 @@ class PermissionsPromoterTest < ActiveSupport::TestCase
 
       users = [first_non_gds_user, second_non_gds_user].each(&:reload)
 
-      assert users.all? do |user|
-        user.role == Roles::OrganisationAdmin.role_name
+      users.each do |user|
+        assert user.role == Roles::OrganisationAdmin.role_name
```

Comment:
I think this relates to the parsing of the `do ... end` block. There are a couple of alternative ways to fix this: (a) use parentheses around the argument being passed to `#assert`; or (b) use braces instead of `do ... end`.

```ruby
assert(
  users.all? do |user|
    user.role == Roles::OrganisationAdmin.role_name
  end
)
```

```ruby
assert users.all? { |user| user.role == Roles::OrganisationAdmin.role_name }
```

However, I think what you've done is probably clearer than either of these anyway.

In fact, given there are only ever two users, I wonder whether it might be clearer to do away with the block altogether and just have two assertions, one for `first_non_gds_user` and another for `second_non_gds_user`...?

Reply:
Good shout. I've made this change in a new commit titled "Simplify assertions in permissions_promoter_test".